source/index.html

```diff
@@ -70,7 +70,16 @@ <h1 class="title">doe sangue</h1>
 			<i id="js-open-menu" class="fa fa-bars" aria-hidden="true"></i>
 		</div>
 	</header><!-- end header -->
-	<section class="call"></section>
+	<section class="call">
+		<article class="call-description">
+			<h2 class="title">Torne-se um doador de sangue</h2>
```

Comment:
The title is very big i think.. @Guilherme-Chinaglia what you say about??

![screenshot-localhost-3000-2017-11-02-13-53-58-736](https://user-images.githubusercontent.com/4200487/32328937-c6492ef6-bfdb-11e7-98c3-ed4f2d0be0e9.png)

Reply:
The ideia is to call attention from people, it's a section that you can put an image in background or change the title, make a campaign. If we try to reduce the title maybe it will be empty at the side. But, what size you suggest to put?

Reply:
I understand the idea here.. What i'm trying to say is maybe to keep the title just in one line, like 
"Torne-se um doador de sangue" instead of:

"Torne-se um
doador de
sangue"

for example..

Reply:
Ok, i understood, give a look, and say what you think.

Reply:
Looks better now @Guilherme-Chinaglia :up: